cli/init.js

```diff
@@ -1,40 +1,136 @@
 const fs = require('fs')
 const path = require('path')
 const chalk = require('chalk')
+const inquirer = require('inquirer')
 const invariant = require('./invariant')
 const { SERVICE_WORKER_BUILD_PATH } = require('../config/constants')
 
-const cwd = process.cwd()
+const CWD = process.env.INIT_CWD || process.cwd()
```

Comment:
`process.env.INIT_CWD` is an environmental variable exposed by NPM that indicates the parent project's CWD when running scripts (i.e. `postinstall`). `process.cwd()` in a `postinstall` script equals the _library's_ directory, not the parent project.